src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -58,7 +58,9 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
         <div className="ServiceList-Title">
           <div className="component-label">
             Workload{' '}
-            {!workload.istioSidecar && <MissingSidecar style={{ marginLeft: '10px' }} tooltip={true} text={''} />}
+            {!workload.istioSidecar && (
+              <MissingSidecar namespace={namespace} style={{ marginLeft: '10px' }} tooltip={true} text={''} />
```

Comment:
Why is the tooltip text blank? Otherwise, we don't even need a tooltip right?

Reply:
I have no idea, it could probably be removed but it's not something I changed for this PR so I don't want to touch it.